ci/scripts/backwards-compat-test.sh

```diff
@@ -25,10 +25,8 @@ shift $((OPTIND -1))
 # profile is either ci-dev or ci-release
 if [[ "$profile" == "ci-dev" ]]; then
   echo "Running in ci-dev mode"
-  ENABLE_RELEASE_PROFILE=false
-elif [[ "$profile" != "ci-release" ]]; then
+elif [[ "$profile" == "ci-release" ]]; then
   echo "Running in ci-release mode"
-  ENABLE_RELEASE_PROFILE=true
```

Comment:
Can just remove this.
We pull the binaries from either our `releases` OR from buildkite.
Either way, these binaries are stored in `target/debug`, and we should always just launch the cluster with that as the target for simplicity.